Jenkinsfile

```diff
@@ -257,7 +258,7 @@ node('fedora-atomic') {
 
                     // Check if a new ostree image compose is needed
                     checkLastImage("${current_stage}")
-                    if (fileExists("${env.WORKSPACE}/NeedNewImage.txt")) {
+                    if (fileExists("${env.WORKSPACE}/NeedNewImage.txt") || ("${env.GENERATE_IMAGE}" == "true")) {
                         stage('ci-pipeline-ostree-image-compose') {
```

Comment:
maybe this if should be inside the stage? That way when looking at stage view we will also see the correct numbers of stages?